src/hastings_vacuum.erl

```diff
@@ -34,13 +34,15 @@
 
 -export([
     handle_db_event/3,
-    clean_db/1
+    clean_db/2
 ]).
 
 
 -record(st, {
-    queue = queue:new(),
-    cleaner
+    qSelectiveClean = queue:new(),
```

Comment:
Big nope to hungarian notation and camel case. Call these selective_clean and full_clean.